go.mod

```diff
@@ -1,6 +1,7 @@
 module contrib.go.opencensus.io/exporter/prometheus
 
 require (
-	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829 // TODO: change the version to v0.9.2 once oc-go version is updated.
+	github.com/prometheus/client_golang v0.9.2
 	go.opencensus.io v0.21.0
+
```

Comment:
Spurious newline. Was this manually added? If so, in the future, you can do
```go
GO111MODULE=on go get -u -v github.com/prometheus/client_golang@v0.9.2
```
and then `go mod tidy`

Reply:
Removed, seems to be added by accident.